aspnetcore/fundamentals/websockets.md

```diff
@@ -0,0 +1,98 @@
+---
+title: WebSockets support in ASP.NET Core | Microsoft Docs
+author: tdykstra
+description: What is WebSockets support in ASP.NET Core and how to use it.
+keywords: ASP.NET Core, WebSockets
+ms.author: tdykstra
+manager: wpickett
+ms.date: 03/25/2017
+ms.topic: article
+ms.assetid: 0e0fedcd-a7b4-4479-8ae0-36eab0229d7e
+ms.technology: aspnet
+ms.prod: aspnet-core
+uid: fundamentals/websockets
+---
+
+# Introduction to WebSockets in ASP.NET Core
+
+By [Tom Dykstra](https://github.com/tdykstra) and [Andrew Stanton-Nurse](https://github.com/anurse)
+
+This article explains how to get started with WebSockets in ASP.NET Core. [WebSocket](https://en.wikipedia.org/wiki/WebSocket) is a protocol that enables two-way persistent communication channels over TCP connections. It is used for applications such as chat, stock tickers, games, anywhere you want real-time functionality in a web application.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/websockets/sample). See the [Next Steps](#next-steps) section for more information.
+
+
+## Prerequisites
+
+* ASP.NET Core 1.1 (does not run on 1.0)
+* Any OS that ASP.NET Core runs on:
+  
+  * Windows 7 / Windows Server 2008 and later
+  * Linux
+  * macOS
+
+   Exception: If your app runs on Windows with IIS, or with WebListener, you must use Windows 8 / Windows Server 2012 or later
+
+* For supported browsers, see http://caniuse.com/#feat=websockets.
+
+## When to use it
+
+Use WebSockets when you need to work directly with a socket connection. A typical reason is a need for best possible performance for real-time games.
```

Comment:
consider something like
Use WebSockets when you need to work directly with a socket connection. You typically work directly with a socket connection when you need the best possible performance, for example, in a real-time game.

I don't really like my version, it's too long, but better connected. The following is too long but gets across my intent:

Use WebSockets when you need to work directly with a socket connection ( typically to get the best possible performance). For example, in real-time games.